tests/smoke/Commands/compile.smoke.yaml

```diff
@@ -11,6 +11,18 @@ tests:
         JUVIX_FILE
     exit-status: 0
 
+  - name: hello-world-no-arg
+    command:
+      shell:
+        - bash
+      script: |
+        cd ./examples/milestone/HelloWorld
+        juvix compile
+        ./HelloWorld
+    exit-status: 0
+    stdout: |
+      hello world!
+
```

Comment:
Can you please add a test to see the error in the absence fo the main field in the juvix. yaml?

Reply:
done